core/templates/dev/head/exploration_editor/EditorServices.js

```diff
@@ -2396,7 +2396,9 @@ oppia.factory('editorFirstTimeEventsService', ['siteAnalyticsService',
     var alreadyRegisteredEvents = {};
     return {
       initRegisterEvents: function(expId) {
-        shouldRegisterEvents = true;
+        if (location.search.split('?')[1] === 'new') {
```

Comment:
Think it's worth dropping 'new' from the url at this point so that if the user refreshes the page, it's not still counted as new?